client/X11/xf_cliprdr.c

```diff
@@ -1097,6 +1104,44 @@ static BOOL xf_cliprdr_process_selection_notify(xfClipboard* clipboard,
 	}
 }
 
+#if defined(WITH_FUSE2) || defined(WITH_FUSE3)
+static xfCliprdrFuseInode* xf_cliprdr_fuse_create_root_node(void)
+{
+	xfCliprdrFuseInode* rootNode = (xfCliprdrFuseInode*)calloc(1, sizeof(xfCliprdrFuseInode));
+	if (!rootNode)
+		return NULL;
+
+	rootNode->ino = 1;
+	rootNode->parent_ino = 1;
+	rootNode->st_mode = S_IFDIR | 0755;
+	rootNode->name = _strdup("/");
```

Comment:
Can't this return null on allocation failure?

Reply:
yes, it could. and therefore it is checked a few lines below.

Reply:
Ah, right. Didn't realize that, sry.